docs/cli/cli-spans.md

```diff
@@ -0,0 +1,38 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+operations to attributes on a Span.
+
+The **span name** SHOULD be of the format `{process.executable.name}`. In case this is
+not informative enough because of interpreters calling other files, e.g. `python script.py` or in
+cases of subcommands, e.g. `flask run mycommand` instrumentation MAY provide a more
+descriptive span name of the format `{process.command_line}` if appropriate.
+
+> **Warning**
+> The `process.command_line` value can have high cardinality.
+
+<!-- semconv span.cli -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`process.command_args`](/docs/attributes-registry/process.md) | string[] | All the command arguments (including the command/executable itself) as received by the process. On Linux-based systems (and some other Unixoid systems supporting procfs), can be set according to the list of null-delimited strings extracted from `proc/[pid]/cmdline`. For libc-based executables, this would be the full argv vector passed to `main`. | `["cmd/otecol", "--config=config.yaml"]` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`process.command_line`](/docs/attributes-registry/process.md) | string | The full command used to launch the process as a single string representing the full command. On Windows, can be set to the result of `GetCommandLineW`. Do not set this if you have to assemble it just for monitoring; use `process.command_args` instead. | `C:\cmd\otecol --config="my directory\config.yaml"` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
the command line and command args are the same, but in a different format. Why specify both?

Reply:
Removed the stringified one, most languages follow C argv like interface.